corehq/apps/hqcase/api/updates.py

```diff
@@ -199,10 +226,9 @@ def _get_bulk_updates(domain, all_data, user):
     errors = []
     for i, data in enumerate(all_data, start=1):
         try:
-            is_creation = data.pop('create', None)
-            if is_creation is None:
+            if 'create' not in data:
                 raise UserError("A 'create' flag is required for each update.")
-            updates.append(_get_individual_update(domain, data, user, is_creation))
+            updates.append(_get_individual_update(domain, data, user, is_creation=True))
```

Comment:
I think I prefer either
```suggestion
            updates.append(_get_individual_update(domain, data, user, is_creation=data.pop('create')))
```
or making the other caller of `_get_individual_update` insert `create` into `data`, so there's only one way to pass it in.